content/community/fiscalhost.md

```diff
@@ -6,20 +6,58 @@ Url: community/fiscalhost
 The XSF is a U.S. 501(c)(3) organization.
 It sometimes acts as a fiscal host for open source projects related to XMPP.
 
-The XSF may choose to accept organizations at the discretion of the board,
-treasurer, or anyone else to whom the board has delegated this authority.
-Projects must be open source with source code, documentation, resources, and
-other non-code assets released using an OSI approved license
-(https://opensource.org/licenses/alphabetical) or a Creative Commons license
-(https://creativecommons.org/licenses/).
-Trademarks will continue to be held by the project and the XSF will not take
-ownership of any project assets.
-The XSF will not charge a fee for this service.
+The XSF (the "sponsor") may choose to accept organizations (the "sponsee") at
```

Comment:
In danger of nitpicking/bike-shedding, is "sponsee" the commonly used term here? I've mainly seen this as "beneficiary", but this might be an American vs. British English thing.

Reply:
I have no idea, but if you have a fiscal sponsor then the opposite would naturally be a sponsee.

Reply:
Let's us `sponsored organization`, throughout.